cms/utils/plugins.py

```diff
@@ -20,7 +20,7 @@
 logger = logging.getLogger(__name__)
 
 
-@lru_cache(maxsize=None)
+@cache
```

Comment:
There is a method just next to it that still uses `lru_cache`, let's port that to `cache` too?

Reply:
@vinitkumar `@lru_cache` without parameters is not equivalent to `@cache`. It has a default `maxsize`